02-intermediate-ruby/exercises/hotel-revisited.md

```diff
@@ -126,9 +126,14 @@ Now that we've got you thinking about design, spend some time to revisit the cod
 - Does this class take on any responsibility that should be delegated to "lower level" classes?
 - Is there code in other classes that directly manipulates this class's instance variables?
 
+You might recall writing a file called `refactor.txt`. Take a look at the refactors that you wrote, and consider the following:
```

Comment:
I would suggest changing "refactors that you wrote" to "refactor plans that you wrote".